enclave-runtime/src/initialization.rs

```diff
@@ -370,20 +367,12 @@ pub(crate) fn init_direct_invocation_server(server_addr: String) -> EnclaveResul
 	let cert =
 		ed25519_self_signed_certificate(signing, "Enclave").map_err(|e| Error::Other(e.into()))?;
 
-	//write certificate and private key pem file
+	//Serialize certificate and private key pem
```

Comment:
Minor style comment (was already wrong before 🙈 )
```suggestion
	// Serialize certificate and private key pem.
```

Reply:
Thank you :+1: , fixed in https://github.com/integritee-network/worker/pull/1034/commits/bb3e8c4b4c06a60c1ab4ea6d50fa14457e206d63